dimod/constrained.py

```diff
@@ -442,6 +444,80 @@ def add_variable(self, v: Variable, vartype: VartypeLike):
         else:
             return self.variables._append(vartype, v)
 
+    @classmethod
+    def from_bqm(cls, bqm: BinaryQuadraticModel) -> 'ConstrainedQuadraticModel':
+        """Alias for :meth:`from_quadratic_model`."""
+        return cls.from_quadratic_model(bqm)
+
+    @classmethod
+    def from_discrete_quadratic_model(cls, dqm: DiscreteQuadraticModel, *,
+                                      relabel_func: Callable[[Variable, int], Variable] = lambda v, c: (v, c),
```

Comment:
Might be a rare case, with no practical/obvious abuse scenario that comes to mind, but function objects technically are mutable (so shouldn't be used for default argument values).

Reply:
Sure, technically. But as you say, I really cannot think of an abuse scenario is this is clear and simple.